project/Build.scala

```diff
@@ -56,6 +56,9 @@ object ScalikeJDBCProjects extends Build {
       "-sourcepath", (baseDirectory in LocalRootProject).value.getAbsolutePath,
       "-doc-source-url", s"https://github.com/scalikejdbc/scalikejdbc/tree/${gitHash}€{FILE_PATH}.scala"
     ),
+    mappings in (Compile, packageSrc) ++= (managedSources in Compile).value.map{ f =>
```

Comment:
Thanks! Would you please write some comment to explain the purpose of the code here? e.g. `// to merge generated sources into sources.jar as well`